modules/candidate_list/jsx/candidateListIndex.js

```diff
@@ -0,0 +1,363 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import Loader from 'Loader';
+import FilterableDataTable from 'FilterableDataTable';
+import Modal from 'Modal';
+
+import OpenProfileForm from './openProfileForm';
+
+/**
+ * Candidate List
+ *
+ * Main module component rendering the candidate list and open profile form
+ *
+ * @author Dave McFarlane
```

Comment:
I don't know who this guy is or how he became an author.

Reply:
technically, you started the work and created that file . I just polished it so did not feel like adding me as an author. ;) I can remove your name if you prefer and make it LORIS team generic (is there such a thing in the LORIS code? A LORIS team generic author by default?).

Reply:
Can you just fix the spelling of it?

Reply:
@driusan so sorry for the typo. It is fixed now.